uploadComplete.php

```diff
@@ -996,13 +996,13 @@
 
 <table class="headerTable">
 <tr><td>
-<div class="headerText">Status</div>
+<div class="headerText"><?php echo _("Status");?></div>
 	<br />
-    <p>File archived as <?php echo $Base_URL . 'archive/' . $uploadedFilename; ?>.</p>
-    <p>Log file available at: <a href='<?php echo $Base_URL . $logfile; ?>'><?php echo $Base_URL . $excelfile; ?></a>.</p>
-    <p>Process completed.  <?php echo $mailOutput; ?></p>
+    <p><?php echo _("File archived as").' '.$Base_URL . $uploadedFile; ?>.</p>
+    <p><?php echo _("Log file available at:");?> <a href='<?php echo $Base_URL . $logfile; ?>'><?php echo $Base_URL . $excelfile; ?></a>.</p>
+    <p><?php echo _("Process completed.")."  ".$mailOutput; ?></p>
     <br />
-    Summary:<?php echo $rownumber . " titles processed.<br />" . nl2br($logSummary); ?><br />
+    <?php echo _("Summary:").'  '.$rownumber . _(" titles processed.")."<br />" . nl2br($logSummary); ?><br />

```

Comment:
we should/could even use ngettext : http://php.net/manual/en/function.ngettext.php
(otherwise, yes, sprintf could do the job)

Long term thought: in fact, we should use a PHP framework, that handle this problem, and separate M/V/C ...